padloper/scripts/init_simple-db.py

```diff
@@ -1,141 +1,82 @@
 from datetime import datetime
-from structure import *
+from padloper.structure import *
```

Comment:
The way `__init.py__` is set up, I believe you can also just do `from padloper import *`.